docs/concepts/storage.md

```diff
@@ -44,13 +44,14 @@ Current options for deployment storage blocks include:
 
 | Storage | Description | Required Library | 
 | --- | --- | --- |
-| [Local File System](/api-ref/prefect/filesystems/#prefect.filesystems.LocalFileSystem) | Store data in a run's local file system. | |
-| [Remote File System](/api-ref/prefect/filesystems/#prefect.filesystems.RemoteFileSystem) | Store data in a any filesystem supported by [`fsspec`](https://filesystem-spec.readthedocs.io/en/latest/). | |
-| [AWS S3 Storage](/api-ref/prefect/filesystems/#prefect.filesystems.S3) | Store data in an AWS S3 bucket. | [`s3fs`](https://s3fs.readthedocs.io/en/latest/) |
-| [Azure Storage](/api-ref/prefect/filesystems/#prefect.filesystems.Azure) | Store data in Azure Datalake and Azure Blob Storage. | [`adlfs`](https://github.com/fsspec/adlfs) |
-| [GitHub Storage](/api-ref/prefect/filesystems/#prefect.filesystems.GitHub) | Store data in a GitHub repository. | |
-| [Google Cloud Storage](/api-ref/prefect/filesystems/#prefect.filesystems.GCS) | Store data in a Google Cloud Platform (GCP) Cloud Storage bucket. | [`gcsfs`](https://gcsfs.readthedocs.io/en/latest/) |
-| [SMB](/api-ref/prefect/filesystems/#prefect.filesystems.SMB) | Store data in SMB shared network storage. | [`smbprotocol`](https://github.com/jborean93/smbprotocol) |
+| [Local File System](/api-ref/prefect/filesystems/#prefect.filesystems.LocalFileSystem) | Store code in a run's local file system. | |
```

Comment:
FYI Terry: All I did here was change "Store data" to "Store code" + added GitLab